frontend/src/metabase/dashboard/components/DashCard/DashCardMenu/DashCardMenu.tsx

```diff
@@ -35,23 +38,20 @@ interface OwnProps {
   visualizationSettings?: VisualizationSettings;
 }
 
-interface TriggerProps {
-  open: boolean;
+export type DashCardMenuItem = {
+  iconName: IconName;
+  label: string;
   onClick: () => void;
-}
-
-interface DispatchProps {
-  onEditQuestion: (question: Question) => void;
-  onDownloadResults: (opts: DownloadQueryResultsOpts) => void;
-}
+  disabled?: boolean;
+} & MenuItemProps;
 
-type DashCardMenuProps = OwnProps & DispatchProps;
-
-const mapDispatchToProps: DispatchProps = {
+const mapDispatchToProps = {
   onEditQuestion: editQuestion,
-  onDownloadResults: downloadQueryResults,
 };
 
+const connector = connect(null, mapDispatchToProps);
```

Comment:
Since the below is already a functional component, could we replace the connector here with `useDispatch` and `dispatch(editQuestion(...))` instead?

Reply:
oh yeah lol